### PR TITLE
Adding mobile navbar css and javascript

### PR DESCRIPTION
**Pull request by zabeluna**
## Description
- CSS, JavaScript and changes to the HTML were added to create a mobile menu.

## What was done?
- The responsive hamburger menu was created using CSS.
- The responsive menu was configured with JavaScript to allow user interaction.

## Notes
- A new div for the logo was implemented to improve responsiveness and allow for switching between desktop and mobile.

### CODE REVIEW

File: assets/styles/desktop/navbar.css [hunk 1]
```diff
@@ -59,3 +59,138 @@
...
+  transition: 120ms;
+}
+
+.nav-link:hover {
+  background: #adadad88;
+}
+
+@media (max-width: 768px) {
+  #hamburger-toggle {
+    display: block;
```
Comment (pedrorosa-dev) [inline assets/styles/desktop/navbar.css:137]: Ficou perfeito o css 😆

File: assets/styles/desktop/navbar.css [hunk 2]
```diff
@@ -59,3 +59,138 @@
 .nav-link:hover {
   background: #adadad88;
 }
+
+#hamburger-toggle {
+  display: none;
+}
+/* barra*/
+.nav-bar {
```
[the remark this answers is not in the document]
Comment (pedrorosa-dev) [reply assets/styles/desktop/navbar.css:67]: E os commits faltou os prefixos.

**Review by pedrorosa-dev — COMMENTED** [review]
Comentarios finalizados

File: assets/styles/mobile/mobile-nav-bar.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const navButton = document.querySelector("#hamburger-toggle");
```
[the remark this answers is not in the document]
Comment (zabeluna) [reply assets/styles/mobile/mobile-nav-bar.js:1]: Obrigada!

File: assets/styles/desktop/navbar.css (hunk 1 above)
Comment (zabeluna) [reply assets/styles/desktop/navbar.css:137]: Muito obrigada!

**Review by pedrorosa-dev — CHANGES_REQUESTED** [review]
Pedido de Alteração: Mexer na nav bar

**Review by pedrorosa-dev — APPROVED** [review]

**Review by pedrorosa-dev — CHANGES_REQUESTED** [review]
"Change Request: Adjustments to the Navbar"

**Review by pedrorosa-dev — APPROVED** [review]

**Pull request merged**